pkg/apis/postgres-operator.crunchydata.com/v1beta1/crunchy_bridgecluster_types.go

```diff
@@ -51,6 +51,11 @@ type CrunchyBridgeClusterSpec struct {
 
 	// The cloud provider where the cluster is located.
 	// Currently Bridge offers aws, azure, and gcp only
+	// ---
+	// Kubernetes assumes the evaluation cost of an enum value is very large.
+	// TODO(k8s-1.29): Drop MaxLength after Kubernetes 1.29; https://issue.k8s.io/119511
+	// +kubebuilder:validation:MaxLength=10
```

Comment:
:thinking: Just curious, why are some 10 and others 15? In both cases it seems to be several times larger than the number of enums listed.

Reply:
Oh, this is the max length of the _values_. The longest value on this field is `azure`, 5 characters, but I went higher so we wouldn't have to babysit the exact counts.